modules/persistent-storage-csi-manila-limitations.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+//
+:_content-type: CONCEPT
+
+[id="persistent-storage-csi-manila-limitations_{context}"]
+= Manila CSI Driver Operator limitations
+
+The following limitations apply to the Manila Container Storage Interface (CSI) Driver Operator:
+
+* Only NFS is supported
++
+OpenStack Manila supports many Network Attached Storage protocols, such as NFS, CIFS, and CEPHFS, and these can be selectively enabled in the OpenStack cloud. The Manila CSI Driver Operator in {product-title} only supports using the NFS protocol. If NFS is not available and enabled in the underlying OpenStack cloud, you cannot use the Manila CSI Driver Operator to provision storage for {product-title}.
+
+* KCS and scale warning
++
+In Red Hat OpenStack Platform 16.1, the Shared File Systems service (Manila) with CephFS through NFS fully supports serving shares to {product-title} through the Manila CSI. However, this solution is not intended for massive scale. Be sure to review important recommendations at https://access.redhat.com/articles/6667651.
+
+* Snapshots are not supported if the back end is CephFS-NFS
```

Comment:
The theory here is that the OpenStack cloud administrator can selectively turn off features even when the underlying storage system/driver support the feature

Reply:
@gouthampacha do we need to even have this statement at all then, since this is generically true rather than specifically true for Manila. If we want to put this generic statement somewhere, we could add it here as a footnote in this table: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/storage/using-container-storage-interface-csi#csi-drivers-supported_persistent-storage-csi. This table shows snapshot support info for all drivers. Your thoughts?